app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,170 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
```

Comment:
Color `#F2F2F2` should be written as `#f2f2f2`<br>Name of variable `light_gray` should be written in all lowercase letters with hyphens instead of underscores